hazelcast-jet-distribution/src/root/config/jvm.options

```diff
@@ -0,0 +1,49 @@
+# JVM Configuration
+
+# One option per line
+# Lines starting with # and empty lines are ignored
+
+# Memory configuration
+#-Xms4g
+#-Xmx4g
+#-XX:+AlwaysPreTouch
+
+# Generate heap dump on out-of-memory error
+# Uncomment to enable
+#-XX:+HeapDumpOnOutOfMemoryError
+#-XX:HeapDumpPath=./
+
+# Enable debugger
+#-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
+
+# Enable JMX
+#-Dcom.sun.management.jmxremote
+#-Dcom.sun.management.jmxremote.port=9010
+#-Dcom.sun.management.jmxremote.authenticate=false
+#-Dcom.sun.management.jmxremote.ssl=false
+#-Dcom.sun.management.jmxremote.local.only=false
+
+# GC configuration
+
+# G1GC Configuration
+#-XX:+UseG1GC
+#-XX:MaxGCPauseMillis=50
+
+# ZGC
+#-XX:+UseZGC
+
+# Shenandoah
+#-XX:+UseShenandoahGC
+
+## JDK 8 GC logging
+#-XX:+PrintGCDetails
+#-XX:+PrintGCDateStamps
+#-XX:+PrintTenuringDistribution
+#-XX:+PrintGCApplicationStoppedTime
+#-Xloggc:logs/gc.log
```

Comment:
will this use JET_HOME by default?

Reply:
Good point,

It would use the current directory, our tutorials expect that it is same as JET_HOME, but users may well be in the bin directory, or elsewhere.

I have added evaluation of the jvm.options file so we can use $JET_HOME there.